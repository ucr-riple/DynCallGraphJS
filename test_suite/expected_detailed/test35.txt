{
    'global (test35.js:17:9:17:12)': [
        'anon (test35.js:14:9:16:2)'
    ],
    'global (test35.js:18:1:18:21)': [
        'anon (test35.js:6:3:8:4)'
    ]
}