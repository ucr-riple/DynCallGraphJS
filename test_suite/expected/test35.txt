{
  '(test35.js:17:9:17:12)': [
    '(test35.js:14:9:16:2)'
  ],
  '(test35.js:18:1:18:21)': [
    '(test35.js:6:3:8:4)'
  ],
  '(test35.js:7:5:7:42)': [ 'log (Native)' ]
}